Schedule_app/models.py

```diff
@@ -1,23 +1,36 @@
 from django.db import models
-from basepage.models import Movie, Cinema, CinemaHall, City
+from basepage.models import Movie, Cinema, CinemaHall, City, SeatModel
+from django.contrib.auth.models import User
+from django.core.validators import validate_comma_separated_integer_list
+
+
+
 # Create your models here.
 
 
 class ScheduledMovies(models.Model):
     start_date = models.DateField()
     end_date = models.DateField()
-    movie = models.ForeignKey(Movie, on_delete=models.SET_NULL, null=True)
+    movie = models.ForeignKey(Movie, on_delete=models.SET_NULL, null=True, related_query_name='moviepass')
 
     def __str__(self):
         return self.movie.title
 
 
 class ScheduleMovieCinema(models.Model):
-    movie = models.ForeignKey(ScheduledMovies, on_delete=models.SET_NULL, null=True)
-    city = models.ForeignKey(City, on_delete=models.SET_NULL, null=True)
-    cinema = models.ForeignKey(Cinema, on_delete=models.SET_NULL, null=True)
-    hall = models.ForeignKey(CinemaHall, on_delete=models.SET_NULL, null=True)
+    movie = models.ForeignKey(ScheduledMovies, on_delete=models.SET_NULL, null=True, related_query_name='movieback')
+    city = models.ForeignKey(City, on_delete=models.SET_NULL, null=True, related_query_name='cityback')
+    cinema = models.ForeignKey(Cinema, on_delete=models.SET_NULL, null=True, related_query_name='cinemaback')
```

Comment:
I still see no changes here. You still have references for all `city`, `cinema` and `hall`